python/graphstorm/eval/eval_func.py

```diff
@@ -195,13 +195,17 @@ def eval_roc_auc(logits,labels):
     # The roc_auc_score function computes the area under the receiver operating characteristic
     # (ROC) curve, which is also denoted by AUC or AUROC. The following returns the average AUC.
     rocauc_list = []
-    labels=labels_to_one_hot(labels, predicted_labels.shape[1])
-    for i in range(labels.shape[1]):
-        # AUC is only defined when there is at least one positive data.
-        if np.sum(labels[:, i] == 1) > 0 and np.sum(labels[:, i] == 0) > 0:
-            is_labeled = labels[:, i] == labels[:, i]
-            rocauc_list.append(roc_auc_score(labels[is_labeled, i],
-                                             predicted_labels[is_labeled, i]))
+    if predicted_labels.shape[1] == 2:
```

Comment:
Can you add comments about in which case `predicted_labels.shape[1] == 2` ?
Do you mean this is binary classification?